lib/generation/generators/visual-widgets/pb_text_gen.dart

```diff
@@ -27,8 +27,8 @@ class PBTextGen extends PBGenerator with PBColorMixin {
             .write(('\'${source.text?.replaceAll('\n', ' ') ?? ''}\'') + ',\n');
       }
       buffer.write('style: ');
-      if(SN_UUIDtoVarName.containsKey('${source.UUID}_textStyle')){
-        buffer.write(SN_UUIDtoVarName[source.UUID + '_textStyle']);
+      if (SN_UUIDtoVarName.containsKey('${source.UUID}_textStyle')) {
+        buffer.write(SN_UUIDtoVarName[source.UUID + '_textStyle'] + ' ?? ');
```

Comment:
I think this might break the optional parameters being passed in as those don't append _XXXXX where _XXXX is the type of the variable/class